src/preferenceManager.cpp

```diff
@@ -45,6 +45,7 @@ void PreferencesManager::save(string filename)
         fprintf(f, "# Empty Epsilon Settings\n# This file will be overwritten by EE.\n\n");
         fprintf(f, "# Include the following line to enable an experimental http server:\n# httpserver=8080\n\n");
         fprintf(f, "# For possible hotkey values check: http://www.sfml-dev.org/documentation/2.3.1/classsf_1_1Keyboard.php#acb4cacd7cc5802dec45724cf3314a142\n\n");
+        fprintf(f, "# Values for ship_window_caption are: spacedust:1, headings:2, callsigns:4 \n# Add them for any combination. Example: ship_window_caption=7 for all three\n\n");
```

Comment:
I would call it "window_flags" instead of "window_caption", as a "caption" is a "a title or brief explanation accompanying an illustration, cartoon, or poster"

Reply:
I had another meaning of caption in my mind, that of "A piece of text appearing on screen as subtitle or other part of a film or broadcast." - or videogame, for that matter, as some games has a "caption" option. But you are right, a more general name would be better as it is not exactly the same as a subtitle and also does not cover the space dust